app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -66,31 +66,20 @@
             app:layout_constraintEnd_toEndOf="@+id/add_profile_activity_user_image_view"
             app:srcCompat="@drawable/rounded_white_background_with_shadow" />
 
-          <TextView
-            android:id="@+id/add_profile_activity_required_heading_text_view"
-            style="@style/TextViewStart"
-            android:layout_width="0dp"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="32dp"
-            android:contentDescription="@string/required_fields_content_description"
-            android:text="@{viewModel.requiredTextWithStar}"
-            android:textColor="@color/component_color_add_profile_activity_label_text_color"
-            android:textSize="12sp"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/add_profile_activity_user_image_view" />
-
           <com.google.android.material.textfield.TextInputLayout
             android:id="@+id/add_profile_activity_user_name"
             style="@style/TextInputLayout"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_marginTop="16dp"
-            android:hint="@string/add_profile_input_name"
             app:errorMessage="@{viewModel.nameErrorMsg}"
+            android:hint="@string/add_profile_input_name"
+            app:helperTextEnabled="true"
+            app:helperText="@{viewModel.requiredField}"
+            app:prefixText="@{viewModel.requiredField}"
```

Comment:
Do we still need the field, or can we just reference the string directly here?

Also, why does both helperText and prefixText need to be specified here?

Reply:
We can now reference the string directly here.
Text helper has been removed, as shown in the videos in the pr description above. PTAL